### DISCUSSION TITLE
Since flat config, where do we find files to import and extend from, such as eslint:recommended?

### DISCUSSION THREAD

**Discussion by conor909** [eslint/eslint#17170] [General] [answered]
Previously we include `eslint` in our package json, and then declare `eslint:recommended` under the `extends` prop.  

I'm attempting to do something like the following, but I cant find the recommended config file.

```js
import eslintRecommended from 'eslint/conf/some-path-to-the-recommended-config.js'

export default [
  eslintRecommended,
  {
    ...
  }
]
```
My eslint version is `8.40.0` and does not have the files mentioned in this [StackOverflow post](https://stackoverflow.com/questions/33162047/where-can-i-find-the-rule-definitions-for-eslintrecommended).  I've found sinilar named files in nested folders but its not clear what to use.

[update]

I found `eslint-recommended` in `eslint/packages/js/src/configs/eslint-recommended`, I presume since `js` has a `package.json` it's published, so I can use:

`npm i @eslint/js` &
```js
import eslintRecommended from '@eslint/js/src/configs/eslint-recommended'
```

Should this be better documented somewhere?

**Comment by mdjermanovic** [comment, marked as answer]
Hi @conor909!

It's documented here: https://eslint.org/docs/latest/use/configure/configuration-files-new#using-predefined-configurations

**Comment by conor909** [comment]
Ah yes I found it, thanks!

next trouble is some arnt compatible and you need to use the FlatCompat class
